Bottom: 365867552404a15718d4fd69fc8f04d117401087
Top:    592a445d4bc6795f0b48c03bde42ee378134f75a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-14 18:39:56 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 3468b9b..399b614 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
